packages/kerosene/src/datetime/constants.ts

```diff
@@ -1,15 +1,48 @@
-export const SECOND = 1_000;
+import {
+  daysInWeek,
+  millisecondsInDay,
+  millisecondsInHour,
+  millisecondsInMinute,
+  millisecondsInSecond,
+  monthsInYear,
+} from "date-fns/constants";
 
-export const MINUTE = 60 * SECOND;
+/**
+ * Milliseconds in one second
```

Comment:
I like that we'll get hints now, did take me a minute originally to understand how to use these.